src/hooks/use-field-id/use-field-id.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import getFieldId from '../../utils/get-field-id';
+
+const useFieldId = (id, sequentialId) => {
+  const [internalId, setId] = React.useState(id);
+
+  React.useEffect(() => {
+    setId(getFieldId({ id }, { id: internalId }, sequentialId));
+  }, [id, internalId, setId, sequentialId]);
+
+  return [internalId];
```

Comment:
I would just return a single value, no need for an array

Reply:
I thought the same at first but then felt array is fine as it would be a breaking change to the API otherwise if later we return multiple values.

Reply:
Isn't this hook only used internally?

Reply:
I didn't export the hook so we can make this change

Reply:
[here](https://github.com/commercetools/ui-kit/pull/999/commits/046438d27b716566d8be8c619903f09317b4f8ff)